.github/ISSUE_TEMPLATE/1.bug.yml

```diff
@@ -26,9 +26,7 @@ body:
       description: |
         **⚠️ Issues that we can't reproduce can't be fixed.**
 
-        If you don't have one, you can use one of these options:
-          - [DataGrid codesandbox template](https://codesandbox.io/s/github/mui/mui-x/tree/master/templates/x-data-grid?file=/src/demo.tsx)
-          - Fork any of the examples in our [documentation](https://mui.com/x/introduction/) by [clicking on the codesandbox or stackblitz icon](https://mui.com/static/docs/forking-an-example.png)
+        Please provide a link to a live example and an unambiguous set of steps to reproduce this bug. See our [documentation](https://mui.com/x/introduction/support/#bug-reproductions) on how to build a reproduction case.
```

Comment:
I think having the links directly in the issue template is faster, there is no need to click & load another page. It's also nice to have the code template because it doesn't require to search through the docs, users can just insert their code directly.

Reply:
> users can just insert their code directly.

This sandbox to be most helpful needs the user:

- to use the Pro plan
- to use the Data Grid
- to use TypeScript
- to use Material UI

I'm not sure it's so great.

Reply:
The packages for community & premium are installed in the template, we could add commented lines like:

```
// Uncomment the line that applies
// import { DataGrid as DataGrid } from 'x-data-grid'
// import { DataGridPro as DataGrid } from 'x-data-grid-pro'
// import { DataGridPremium as DataGrid } from 'x-data-grid-premium'
```

And MUI & typescript are the most common choices. If I was a user, I feel like I would be more satisfied to have the template available right there rather than to have to go through multiple pages/links. I think the other packages (charts, pickers, tree-view) should also include a link like this one in this section.

Reply:
I have tried an alternative tradeoff, keeping this bug reproduction template.